lib/stripe/stripe_object.rb

```diff
@@ -72,13 +72,17 @@ def refresh_from(values, opts, partial=false)
     #
     # * +values+ - Hash of values to use to update the current attributes of
     #   the object.
+    # * +:opts+ Options for StripeObject like an API key.
     #
     # ==== Options
     #
-    # * +:opts+ Options for StripeObject like an API key.
-    def update_attributes(values, opts = {})
+    # * +:dirty+ Whether values should be initiated as "dirty" (unsaved).
+    #   Defaults to true.
+    def update_attributes(values, opts = nil, options = {})
```

Comment:
Are there names that we could use here other than `opts` + `options`? It's pretty confusing with the 2 similar names imo.